src/repositories/userRepository.js

```diff
@@ -167,6 +168,21 @@ class UserRepository {
       throw new Error(error);
     }
   }
+
+  /**
+     *
+     * @param {obj} where
+     * @returns {obj} record is object if managerEmail found or null if not
+     */
+  async findByAccommodationIdAndRoomId(where) {
```

Comment:
Please rename the `where` parameter to make it more intuitive, e.g `keyword` or `id`